source/client/benchmark_client_impl.h

```diff
@@ -66,6 +67,48 @@ class Http1PoolImpl : public Envoy::Http::Http1::ProdConnPoolImpl {
   bool prefetch_connections_{};
 };
 
+// Vanilla Envoy's HTTP/2 pool is single connection only (or actually sometimes dual in connection
+// drainage scenarios). Http2PoolImpl is an experimental pool, which uses multiple vanilla Envoy
+// HTTP/2 pools under the hood. Using multiple connections is useful when testing backends that need
+// multiple connections to distribute load internally. Combining multiple connections with
+// --max-requests-per-connection may help as well, as doing periodically initiating new connections
+// may help the benchmark target by giving it an opportunity to rebalance.
+class Http2PoolImpl : public Envoy::Http::ConnectionPool::Instance,
+                      public Envoy::Http::ConnPoolImplBase {
+public:
+  Http2PoolImpl(
```

Comment:
Can we document the constructor and its arguments? 

Applies to all other public portions of this API, unless they have been documented in a parent class already.

Reply:
Added a single doc comment pointing those looking for doc comments to the interface/base class